exposed-core/src/main/kotlin/org/jetbrains/exposed/sql/vendors/OracleDialect.kt

```diff
@@ -15,6 +15,7 @@ import java.util.*
 internal object OracleDataTypeProvider : DataTypeProvider() {
     override fun byteType(): String = "SMALLINT"
     override fun ubyteType(): String = "NUMBER(4)"
+    override fun shortType(): String = "NUMBER(5)"
```

Comment:
This should probably be labelled as a breaking change, no? Especially if moving from SMALLINT to this is a restrictive change. Although any DDL type changes should probably be clearly listed as a breaking change, whether it's more or less restrictive.